receiver/azureeventhubreceiver/azureresourcelogs_unmarshaler_test.go

```diff
@@ -236,7 +236,7 @@ func TestUnmarshalLogs(t *testing.T) {
 	expectedMinimum := plog.NewLogs()
 	resourceLogs := expectedMinimum.ResourceLogs().AppendEmpty()
 	scopeLogs := resourceLogs.ScopeLogs().AppendEmpty()
-	scopeLogs.Scope().SetName("otelcol/" + typeStr)
+	scopeLogs.Scope().SetName("otelcol/azureeventhubreceiver")
```

Comment:
Should this use `receiverScopeName`?

Reply:
it's a test so the idea is to hardcode things so we fail loudly if things change from under us.

Reply:
fair enough, the previous test was using `typeStr` but i agree that hardcoding is ok here.

Reply:
I'm codeowner of this component and I would like to apply to the maximum pain principle here. I need things to break or I'll happily gloss over them.